LDAP-Auth/LDAPAuthenticationProviderPlugin.cs

```diff
@@ -208,13 +211,89 @@ public Task ChangePassword(User user, string newPassword)
             return Task.CompletedTask;
         }
 
-        private static bool LdapClient_UserDefinedServerCertValidationDelegate(
+        /// <summary>
+        /// Deligate for validating LDAP server cert against a user provided CA.
+        /// </summary>
+        /// <param name="sender">An object that contains state information for this validation.</param>
+        /// <param name="certificate">TLS certificate provided by the server.</param>
+        /// <param name="chain">TLS chain provided by the server.</param>
+        /// <param name="sslPolicyErrors">Bitset of possible policy errors with the server's certificate.</param>
+        /// <returns>True if server cert is valid and trusted by the CA, otherwise false.</returns>
+        private bool LdapClient_UserDefinedServerCertValidationDelegate(
+            object sender,
+            X509Certificate certificate,
+            X509Chain chain,
+            SslPolicyErrors sslPolicyErrors)
+        {
+            if ((sslPolicyErrors & SslPolicyErrors.RemoteCertificateNameMismatch) > 0 ||
+                (sslPolicyErrors & SslPolicyErrors.RemoteCertificateNotAvailable) > 0)
+            {
+                _logger.LogWarning("Provided certificate not valid for remote name");
+                return false;
+            }
+
+            using var rootChain = new X509Chain
+            {
+                ChainPolicy =
+                {
+                    VerificationFlags = X509VerificationFlags.AllowUnknownCertificateAuthority |
+                                        X509VerificationFlags.IgnoreCertificateAuthorityRevocationUnknown |
+                                        X509VerificationFlags.IgnoreCtlNotTimeValid |
+                                        X509VerificationFlags.IgnoreCtlSignerRevocationUnknown |
+                                        X509VerificationFlags.IgnoreEndRevocationUnknown,
```

Comment:
```suggestion
                    VerificationFlags = X509VerificationFlags.AllowUnknownCertificateAuthority
                                        | X509VerificationFlags.IgnoreCertificateAuthorityRevocationUnknown
                                        | X509VerificationFlags.IgnoreCtlNotTimeValid
                                        | X509VerificationFlags.IgnoreCtlSignerRevocationUnknown
                                        | X509VerificationFlags.IgnoreEndRevocationUnknown,
```

Just moving the `|` to start of newline